src/VkBootstrap.cpp

```diff
@@ -1724,27 +1749,25 @@ detail::Result<Swapchain> SwapchainBuilder::build() const {
 	swapchain_create_info.clipped = info.clipped;
 	swapchain_create_info.oldSwapchain = info.old_swapchain;
 	Swapchain swapchain{};
-    PFN_vkCreateSwapchainKHR swapchain_create_proc;
-    detail::vulkan_functions().get_device_proc_addr(info.device, swapchain_create_proc, "vkCreateSwapchainKHR");
-    auto res = swapchain_create_proc(info.device, &swapchain_create_info, info.allocation_callbacks, &swapchain.swapchain);
-;	if (res != VK_SUCCESS) {
+	PFN_vkCreateSwapchainKHR swapchain_create_proc;
+	detail::vulkan_functions().get_device_proc_addr(info.device, swapchain_create_proc, "vkCreateSwapchainKHR");
+	auto res = swapchain_create_proc(
+	    info.device, &swapchain_create_info, info.allocation_callbacks, &swapchain.swapchain);
+	;
```

Comment:
There is a stray semi here (that looks like it came from my previous commit)

Reply:
Thanks, I should do a separate PR that fixes some of the formatting issues I have with the code.